### PR TITLE
Humble release versions 2023-01-10

### PR DESCRIPTION
**Pull request by audrow**
Changes: https://github.com/orgs/ros2/projects/44

CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/17985/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/12512/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/18617/)
* RHEL [Build Status](http://ci.ros2.org/job/ci_linux-rhel/295/)

Packaging:
* https://ci.ros2.org/view/packaging/job/packaging_linux/2910/
* https://ci.ros2.org/view/packaging/job/packaging_linux-aarch64/2263/
* https://ci.ros2.org/view/packaging/job/packaging_linux-rhel/1411/
* https://ci.ros2.org/view/packaging/job/packaging_windows/2740/
* https://ci.ros2.org/view/packaging/job/packaging_windows_debug/1699/

### CODE REVIEW

File: ros2.repos [hunk 1]
```diff
@@ -38,7 +38,7 @@ repositories:
   eProsima/foonathan_memory_vendor:
     type: git
     url: https://github.com/eProsima/foonathan_memory_vendor.git
-    version: v1.2.1
+    version: v1.2.2
```
Comment (clalancette) [inline ros2.repos]: This should be 1.2.0, as that is the packages we are shipping:

```suggestion
    version: v1.2.0
```

**Review by clalancette — APPROVED** [review]
This looks good to me, assuming that CI comes back green.

**Comment by audrow** [comment]
Looks like CI is almost good. It seems like https://github.com/ros2/rclpy/pull/1066 is the outstanding issue.

**Comment by audrow** [comment]
I merged in the rclpy PR, released rclpy, and bumped the version in the ros2.repos file in 80fe3c7.

**Pull request merged**